Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_name</key>
 			<string>passwordElementID</string>
 			<key>pfm_title</key>
-			<string>Password Element ID</string>
+			<string>passwordElementID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>given_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for First Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappingfirstname</string>
+			<key>pfm_name</key>
+			<string>map_firstname</string>
```

Comment:
Unless I'm missing something, I think this should be `oidcmappingfirstname`.

Reply:
Regarding all the comments that had 'oidcmapping' I do believe the ones I have here are correct, but I'll confirm with TwoCanoes. I think those docs are outdated and may have been missed in their updates.